src/reducers/computeRenderedRows.js

```diff
@@ -42,20 +44,25 @@ export default function computeRenderedRows(state, scrollAnchor) {
   const { bodyHeight } = tableHeightsSelector(newState);
   const maxScrollY = scrollContentHeight - bodyHeight;
 
-  // NOTE (jordan) This handles #115 where resizing the viewport may
-  // leave only a subset of rows shown, but no scrollbar to scroll up to the first rows.
-  if (maxScrollY === 0) {
-    if (rowRange.firstViewportIdx > 0) {
-      rowRange = calculateRenderedRowRange(newState, {
-        firstOffset: 0,
-        lastIndex: rowsCount - 1,
-      });
+  // if we are scrolling, only update the offsets for the rows which are
+  // inside the viewport
+  if (state.scrolling) {
```

Comment:
So this is where we branch out the logic.
Only change here is that we call `calculateViewPortRenderedRowOffsets` instead of  `calculateRenderedRowRange` if `scrolling` is true.